UnitTests/UnitTests.UWP/UI/Controls/Test_TokenizingTextBox_AutomationPeer.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Automation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.UI.Xaml.Automation.Peers;
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI.Automation.Peers;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+
+namespace UnitTests.UWP.UI.Controls
+{
+    [TestClass]
+    [TestCategory("Test_TokenizingTextBox")]
+    public class Test_TokenizingTextBox_AutomationPeer : VisualUITestBase
```

Comment:
I think it's worth adding a test ensuring that the actual list of tokens is also being presented to UIA. Currently, we wouldn't notice if this breaks and as such leaving out crucial information to users using UIA.

Reply:
Apologies for the huge delay looking into these comments. I've added the additional test for checking the children of the automation peer.

To select the items, I used the `SelectAllTokensAndText` method on the `TokenizingTextBox` but noted in the test, this will select an empty text item and append it to the collection.